src/plugins/typedispatcher/helpers.c

```diff
@@ -0,0 +1,437 @@
+#include "typehelper.h"
+#include <ctype.h>
+#include <kdbease.h>     //elektraArrayIncName
+#include <kdbmodule.h>   //elektraModulesInit, elektraModulesClose
+#include <kdbos.h>       //elektraNamespace;
+#include <kdbprivate.h>  //elektraPluginClose. elektraPluginOpen, elektraStrNDup
```

Comment:
ok thats really a minor thing but maybe consistent about using , ; and . or nothing at all, otherwise one could maybe believe that there is some meaning behind it